components/mediation-initializer/org.wso2.carbon.mediation.transport.handlers/src/main/java/org/wso2/carbon/mediation/transport/handlers/AbstractHttpGetRequestProcessor.java

```diff
@@ -235,6 +235,25 @@ protected String getServicePath() {
         return servicePath;
     }
 
+    /**
+     * Returns the favicon URL.
+     *
+     * @return favicon URL
+     */
+    protected String getFaviconUrl() {
+        if (faviconUrl == null) {
+            String hostName = "localhost";
+            ServerConfiguration serverConfig = ServerConfiguration.getInstance();
+            if (serverConfig.getFirstProperty("HostName") != null) {
```

Comment:
Let's replace the "Hostname" and "localhost" strings with the constants defined in this file.